lib/msal-browser/test/app/PublicClientApplication.spec.ts

```diff
@@ -87,19 +76,6 @@ describe("PublicClientApplication.ts Class Unit Tests", () => {
     describe("Redirect Flow Unit tests", () => {
 
         describe("handleRedirectCallback()", () => {
```

Comment:
Rename this to handleRedirectPromise, and make sure all tests in this suite are now using `handleRedirectPromise()` instead of `handleRedirectCallback()`

Reply:
Done